### DISCUSSION TITLE
Missing features from Anydesk

### DISCUSSION THREAD

**Discussion by SmartManoj** [rustdesk/rustdesk#3895] [Ideas]
https://support.anydesk.com/Command_Line_Interface

**Comment by SmartManoj** [comment]
https://support.anydesk.com/Keyboard

**Comment by rustdesk** [comment]
Thanks

**Comment by grummbeer** [comment]
one link dead, cli already in mind, right?

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/2865#discussioncomment-4725625
